hack/local-development/start-gardenlet

```diff
@@ -29,19 +31,90 @@ else
   trap cleanup_kubeconfig EXIT
 fi
 
+YQ="$REPO_ROOT/hack/tools/bin/yq"
+GARDEN_KUBECONFIG="${GARDEN_KUBECONFIG:-$kubeconfig}"
+
+# determine the seed to manage; if multiple seeds exist on the
+# garden, the SEED_NAME env variable must be set
+if [ -n "${SEED_NAME:-}" ]; then
+  seedConfig="$(kubectl --kubeconfig="$GARDEN_KUBECONFIG" get seed "$SEED_NAME" -o yaml)"
+else
+  seeds="$(kubectl --kubeconfig="$GARDEN_KUBECONFIG" get seeds -o yaml | $YQ eval '.items' -)"
+  case $(echo "$seeds" | $YQ eval 'length' -) in
+    0)
+      echo "No seed found in garden cluster."
+      exit 1
+      ;;
+    1)
+      seedConfig="$(echo "$seeds" | $YQ eval '.[0]' -)"
+      SEED_NAME="$(echo "$seedConfig" | $YQ eval '.metadata.name' -)"
+      ;;
+    *)
+      names="$(echo "$seeds" | $YQ eval --tojson --indent 0 '[.[].metadata.name]' -)"
+      echo "Multiple seeds exist in the garden, please specify \$SEED_NAME to select one of $names"
+      exit 1
+      ;;
+  esac
+fi
+
+# cleanup a bit to make the generated configuration file look a bit neater
+tmpFile="$DEV_DIR/seed-$SEED_NAME.yaml"
+echo "$seedConfig" | $YQ eval 'del(
+  .apiVersion,
+  .kind,
+  .status,
+  .metadata.annotations,
+  .metadata.managedFields,
+  .metadata.creationTimestamp,
+  .metadata.finalizers,
+  .metadata.generation,
+  .metadata.resourceVersion,
+  .metadata.uid
+)' - > "$tmpFile"
+
+# create seed-specific configuration file if it doesn't exist yet
+configFile="$DEV_DIR/20-componentconfig-gardenlet-seed-$SEED_NAME.yaml"
+
+if [ ! -f "$configFile" ]; then
+  cp "$DEV_DIR/20-componentconfig-gardenlet.yaml" "$configFile"
```

Comment:
Let's copy from the examples dir. Otherwise devs that have a `seedSelector={}` in their `dev/20-com...` (which is currently the default), will be unable to start the gardenlet and it fails with a long message including `either seed config or seed selector is required`.
```suggestion
  cp "$EXAMPLE_DIR/20-componentconfig-gardenlet.yaml" "$configFile"
```


Other option: keep the copy but make sure to remove the `seedSelector` from it.

Reply:
Hmm, I think, this is not resolved.